app/css/application.css

```diff
@@ -81,6 +81,14 @@
     --tabIconColorSelected: var(--c-604FCD);
     --modalBackgroundColor: rgba(251, 252, 254, 0.7);
 
+    /* https://codepen.io/sosuke/pen/Pjoqqp */
+    --notifications-active-filter: invert(76%) sepia(54%) saturate(3406%)
+        hue-rotate(349deg) brightness(92%) contrast(111%);
+    --notifications-inactive-filter: invert(17%) sepia(25%) saturate(368%)
+        hue-rotate(188deg) brightness(93%) contrast(86%);
+    --primary-reputation-filter: invert(88%) sepia(37%) saturate(870%)
+        hue-rotate(355deg) brightness(90%) contrast(96%);
```

Comment:
See https://exercism-team.slack.com/archives/CARRG4MNC/p1615919428002900 for an understanding of this. 
I'll document this in the README or somewhere properly at some stage. I've reproduced the explanation below:

---

Those with an interest in CSS might appreciate this:
1. Take an SVG. Make it black on transparent.
2. Render the SVG using <img> tag in HTML
3. Go to https://codepen.io/sosuke/pen/Pjoqqp, put a color (hex) in and spam "Compute Filters" until the thing at the bottom of the screen says it's close enough (#E2CB2D is good to play with)
4. Take the filter and apply it to the img via CSS.

You now have color-changable icons through CSS while getting all the value of the <img> tag rather than <svg> tag (for context, rendering 100 exercise icons as <svg> takes the browser **20 seconds** in comparison with basically instant as <img>).